src/main/kotlin/gdx/liftoff/data/platforms/IOSMOE.kt

```diff
@@ -146,6 +146,7 @@ class IOSMOEGradleFile(val project: Project) : GradleFile(IOSMOE.ID) {
     dependencies.add("project(':${Core.ID}')")
     addDependency("io.github.berstanio:gdx-backend-moe:\$gdxMoeVersion")
     addSpecialDependency("natives \"com.badlogicgames.gdx:gdx-platform:\$gdxVersion:natives-ios\"")
+    addSpecialDependency("natives \"com.badlogicgames.gdx:gdx-backend-robovm-metalangle:\$gdxVersion\"")
```

Comment:
Cool, metalangle by default. I wonder if we should do this for RoboVM, too, later...

Reply:
https://github.com/libgdx/libgdx/issues/7422
It's planned, yes!